projects/1.16/assets/xaeros-world-map/xaeroworldmap/lang/zh_cn.json

```
@@ -1,57 +1,120 @@
+  "gui.xaero_box_controls_minimap": "\n 路径点控制： \n §2%s§r 创建一个新的路径点。 \n §2%s§r 快速创建一个临时路径点。 \n §2%s§r 切换路径点。 \n §2%s§r 切换渲染所有路径点。 \n §2%s§r 打开完整的路径点菜单。 \n 鼠标右键单击现有路径点以获得更多路径点控制。 \n \n ",
+  "gui.xaero_box_export": "将地图导出为一个 PNG 文件。",
+  "gui.xaero_export_confirm_1": "是否要将地图导出为 PNG 文件？",
+  "gui.xaero_export_confirm_2": "游戏在运行时会暂时冻结。",
+  "gui.xaero_box_map_switching": "地图切换选项",
+  "gui.xaero_box_open_waypoints": "路径点",
+  "gui.xaero_box_close_waypoints": "关闭路径点s",
+  "gui.xaero_box_open_settings": "§2%s§r 打开设置",
+  "gui.xaero_box_close_settings": "§2%s§r 关闭设置",
+  "gui.xaero_wm_up": "[向上]",
+  "gui.xaero_wm_down": "[向下]",
+  "gui.xaero_filter_waypoints_by_name": "筛选路径点",
+  "gui.xaero_wm_search_invalid_regex": "无效的正则表达式语法!",
+  "gui.xaero_box_full_waypoints_menu": "§2%s§r 打开完整的路径点菜单",
+  "gui.xaero_box_rendering_all_sets": "§2%s§r 渲染所有路径点",
+  "gui.xaero_box_rendering_current_set": "§2%s§r 仅渲染当前设置",
+  "gui.xaero_zoom_buttons": "缩放按钮",
+  "gui.xaero_box_showing_disabled": "显示禁用路径点s",
+  "gui.xaero_box_hiding_disabled": "隐藏禁用路径点s",
+  "gui.xaero_box_waypoints_minimap_required": "路径点功能需要 Xaero's Minimap 模组！",
+  "gui.xaero_box_waypoints_disabled": "世界地图设置中路径点被禁用！",
+  "gui.xaero_right_click_waypoint_edit": "§2%s§r 编辑路径点",
+  "gui.xaero_right_click_waypoint_teleport": "§2%s§r 传送到路径点",
+  "gui.xaero_right_click_waypoint_disable": "§2%s§r 禁用路径点",
+  "gui.xaero_right_click_waypoint_enable": "§2%s§r 启用路径点",
+  "gui.xaero_right_click_waypoint_restore": "恢复路径点",
+  "gui.xaero_right_click_waypoint_delete_confirm": "§2%s§r 确认删除",
+  "gui.xaero_right_click_waypoint_delete": "§2%s§r 删除路径点",
+  "gui.xaero_right_click_map_create_waypoint": "§2%s§r 创建路径点",
+  "gui.xaero_right_click_map_create_temporary_waypoint": "§2%s§r 设置临时路径点",
+  "gui.xaero_right_click_map_waypoints_menu": "§2%s§r 打开路径点菜单",
+  "gui.xaero_right_click_map_teleport": "传送到这里",
+  "gui.xaero_right_click_map_title": "选择一个选项",
+  "gui.xaero_right_click_box_map_settings": "§2%s§r 打开设置",
+  "gui.xaero_right_click_box_map_export": "将地图导出为 PNG",
+  "gui.xaero_waypoint_backgrounds": "渲染路径点背景",
+  "gui.xaero_right_click_map_cant_teleport": "§8(传送) 模糊的 Y 坐标",
+  "gui.xaero_right_click_map_cant_create_waypoint": "§8不能在这里设置路径点",
+  "gui.xaero_no_world_map_item_message": "该模组被配置为在您的快捷栏中或装备中需要以下物品：",
+  "gui.xaero_right_click_waypoint_share": "在聊天栏分享路径点",
+  "gui.xaero_right_click_map_share_location": "在聊天栏分享位置",
+  "gui.xaero_right_click_map_cant_share_location": "§8无法分享此位置",
+  "gui.xaero_current_map_locked1": "当前选择的地图被另一个游戏进程（Minecraft 客户端）所使用。",
+  "gui.xaero_current_map_locked2": "请选择该服务器的不同地图（左下角图标）。",
+  "gui.xaero_wm_detect_ambiguous_y": "在发现中检测到不明确的 Y 坐标",
+  "gui.xaero_wm_box_detect_ambiguous_y": "地图上的单个像素可以包含多个透明图层和它们下方的不透明方块。 在较旧的非缓存地图数据文件中，在这种情况下只存储不透明方块的 Y 坐标（高度），这通常会使直接传送到存储的 Y 坐标不安全。 当此设置为 ON 时，模组会将此类地图像素视为未发现的区块。 它默认为 2D 航点并禁用直接传送到此类像素。 禁用此功能需您自担风险！ ",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "gui.xaero_wm_box_detect_ambiguous_y": "地图上的单个像素可以包含多个透明图层和它们下方的不透明方块。在较旧的非缓存地图数据文件中，在这种情况下只存储不透明方块的 Y 坐标（高度），这通常会使直接传送到存储的 Y 坐标不安全。当此设置为 ON 时，模组会将此类地图像素视为未发现的区块。 它默认为 2D 航点并禁用直接传送到此类像素。 禁用此功能需你自担风险！ ",
```